manifests/init.pp

```diff
@@ -29,8 +29,10 @@
   $tomcat = $gutterball::params::tomcat,
 ) inherits gutterball::params {
 
+  $_ = cache_data('gutterball_db_password', $dbpassword)
+  $__ = cache_data('gutterball_db_user', $dbuser)
+  $___ = cache_data('gutterball_tomcat', $tomcat)
```

Comment:
The gutterball module is not top level. We wrap it within the Katello module so that we configure it to our needs. AFAIK Kafo can only access parameters from modules that are declared within the answers file itself.